Composer/packages/client/src/components/CreationFlow/v2/CreateOptions.tsx

```diff
@@ -242,10 +247,19 @@ export function CreateOptionsV2(props: CreateOptionsProps) {
             </ScrollablePane>
           </div>
           <div css={templateDetailContainer} data-is-scrollable="true">
+            {fetchReadMePending && <LoadingSpinner message={''} />}
             <TemplateDetailView readMe={readMe} templateId={currentTemplate} />
           </div>
         </div>
         <DialogFooter>
+          <Link
+            styles={{ root: { fontSize: '12px', float: 'left' } }}
+            href="https://github.com/microsoft/botframework-components/issues/new?assignees=&labels=needs-triage%2C+feature-request&template=-net-sdk-feature-request.md&title="
```

Comment:
pull this out into a constant at the top of the file

Reply:
make sure to add target="_blank" also to this link so that it will open in a new window.

Reply:
Handled